neon_diana_utils/helm_charts/http/stt-fasterwhisper/values.yaml

```diff
@@ -0,0 +1,63 @@
+# Default values for tts-coqui.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+serviceName: backend-fasterwhisper
+replicaCount: 1
+subdomain: fasterwhisper
+domain: diana.local
+servicePort: "8080"
+
+image:
+  repository: docker.io/smartgic/ovos-stt-server-fasterwhisper
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: alpha
+volumeMounts:
+  - name: config-volume
+    mountPath: "/home/ovos/.config/mycroft" # Adjusted for the 'ovos' user
+    readOnly: false
+volumes:
+  - name: config-volume
+    configMap:
+      name: mycroft-config
+      items:
+        - key: mycroft.conf
+          path: mycroft.conf
```

Comment:
Is this using ovos-config? If so, can we set envvars to use `/etc/neon/diana.yaml` to use the existing config file used for other backend services?

Reply:
Yes, ovos-config is installed...let me look up the envvars we need and make the adjustments